pkg/asset/manifests/kube-core-operator.go

```diff
@@ -85,8 +84,7 @@ func (kco *kubeCoreOperator) coreConfig() (*kubecore.OperatorConfig, error) {
 	coreConfig.AuthConfig.OIDCGroupsClaim = authConfigOIDCGroupsClaim
 	coreConfig.AuthConfig.OIDCUsernameClaim = authConfigOIDCUsernameClaim
 
-	svcCidr := kco.installConfig.Networking.ServiceCIDR
-	ip, err := cidr.Host(&net.IPNet{IP: svcCidr.IP, Mask: svcCidr.Mask}, 10)
+	ip, err := cidr.Host(&kco.installConfig.Networking.ServiceCIDR.IPNet, 10)
```

Comment:
I havent traced this back to see if `ServiceCIDR` is inotialed with an IP that doesn't start the subnet.  But if that's the concern you can use [`IP.Mask`][1] to get the initial IP.  Something like:

```go
svcCIDR := kco.installConfig.Networking.ServiceCIDR
ip, err := cidr.Host(&net.IPNet{IP: svcCIDR.IP.Mask(svcCIDR), Mask: svcCIDR.Mask}, 10)
```

[1]: https://godoc.org/net#IP.Mask